apps/app/src/electron/IPCServer.ts

```diff
@@ -290,6 +292,63 @@ export class IPCServer
 			this.callbacks.onAgreeToUserAgreement()
 		}
 	}
+
+	async exportProject(): Promise<void> {
+		const exportData = JSON.stringify(this.storage.getProjectForExport())
```

Comment:
Shouldn't this happen after closing the dialog? If this turns out to be a heavy operation, it means we've done all of this work for nothing.

Reply:
Good point!